blocks/columns/columns.js

```diff
@@ -1,4 +1,16 @@
 export default function decorate(block) {
+  // Selectors for buttons
+  const firstBtn = document.querySelectorAll('.columns .button-container:nth-of-type(2)');
+  const secondBtn = document.querySelectorAll('.columns .button-container:nth-of-type(3)');
+
+  // Add class to each to button container
+  firstBtn.forEach((btn) => {
+    btn.classList.add('first__button');
+  });
+  secondBtn.forEach((btn) => {
+    btn.classList.add('second__button');
```

Comment:
I think you forgot the B(lock) from BEM. `first-button` and `second-button` are E(lement), so you should have something like `column__first-button` instead.